README.md

```diff
@@ -1 +1,43 @@
-# pipefy_message
\ No newline at end of file
+# PipefyMessage
```

Comment:
How this project doesn't have docker, I think is a good idea to specify the version of Ruby that is necessary to run locally.

Reply:
sure, we just have to choose the version that we'll use and then I update the readme

Reply:
I think version 2.6.6

Reply:
ok, I'll create a section for that

Reply:
done